ui/README.md

```diff
@@ -0,0 +1,24 @@
+## Dashboard for Mainflux in Elm
+Dashboard made with [elm-bootstrap](http://elm-bootstrap.info/).
+
+### Install
+```
+git clone https://github.com/mainflux/mainflux
+cd mainflux/ui
+make
```

Comment:
Maybe we should mention that elm needs to be installed and provide a link to installation instructions.

Reply:
Would not hurt...

Reply:
Think it's better to leave it for the master branch and focus on other things now.

Reply:
Umm @darkodraskovic when I think about this, maybe it would be good to add just one sentence here like - "Prerequisite for building Mainflux UI is [Elm](https://elm-lang.org/) - a delightful language for reliable webapps :wink:"